File: web/supermercado/domain/categoria.php
<?php 

	class Categoria {
		private $id;
		private $nombre;



    public function getId()
    {
        return $this->id;
    }

    public function setId($id)
    {
        $this->id = $id;

        return $this;
    }

 
	
		public function getNombre()
		{
				return $this->nombre;
		}

	
		public function setNombre($nombre)
		{
				$this->nombre = $nombre;

				return $this;
		}
}


 ?>

File: movil/SupermercadoApp/app/src/main/java/una/ac/cr/supermercadoapp/domain/Categoria.java
package una.ac.cr.supermercadoapp.domain;

import java.io.Serializable;

public class Categoria implements Serializable {

    private int id;
    private String nombre;
    private int estadosync;

    public int getEstadosync() {
        return estadosync;
    }
    public void setEstadosync(int estadosync) {
        this.estadosync = estadosync;
    }

    public Categoria(String nombre) {
        this.nombre = nombre;
    }

    public Categoria(String nombre, int estadosync) {
        this.nombre = nombre;
        this.estadosync = estadosync;
    }

    public Categoria(int id, String nombre, int estadosync) {
        this.id = id;
        this.nombre = nombre;
        this.estadosync = estadosync;
    }

    public Categoria() {
    }

    public Categoria(int id, String nombre) {
        this.id = id;
        this.nombre = nombre;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}
